scripts/tests.js

```diff
@@ -9,6 +9,6 @@ import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({adapter: new Adapter()});
 
-var testsContext = require.context('.', true, /.spec.[t|j]sx?$/);
+var testsContext = require.context('.', true, /.spec.(js|ts|tsx)$/);
```

Comment:
This means we won't use `.jsx` any more right?

Reply:
Yes, we'll no longer use `.jsx` - the extension, but will continue using `jsx` syntax in `.tsx` files.